.github/workflows/python-package-conda.yml

```diff
@@ -31,18 +32,36 @@ jobs:
 
         # Install test tools
         conda install pip coverage pytest
-        conda install -c conda-forge pytest-cov
+        pip install coveralls
 
         # Install python-control dependencies
         conda install numpy matplotlib scipy
         if [[ '${{matrix.slycot}}' == 'conda' ]]; then
           conda install -c conda-forge slycot
         fi
+
     - name: Test with pytest
       env:
         PYTHON_CONTROL_ARRAY_AND_MATRIX: ${{ matrix.array-and-matrix }}
       run: |
         source $CONDA/bin/activate test-environment
         # Use xvfb-run instead of pytest-xvfb to get proper mpl backend
+        # Use coverage instead of pytest-cov to get .coverage file
```

Comment:
pytest-cov produces one. See python-control/Slycot#140. Use the --cov parameter.